src/constants.asm

```diff
@@ -52,40 +52,42 @@ BUTTON_SELECT := $20
 BUTTON_START := $10
 BUTTON_DPAD := BUTTON_UP | BUTTON_DOWN | BUTTON_LEFT | BUTTON_RIGHT
 
-MODE_TETRIS := 0
-MODE_TSPINS := 1
-MODE_SEED := 2
-MODE_PARITY := 3
-MODE_PACE := 4
-MODE_PRESETS := 5
-MODE_TYPEB := 6
-MODE_FLOOR := 7
-MODE_CRUNCH := 8
-MODE_TAP := 9
-MODE_TRANSITION := 10
-MODE_TAPQTY := 11
-MODE_CHECKERBOARD := 12
-MODE_GARBAGE := 13
-MODE_DROUGHT := 14
-MODE_DAS := 15
-MODE_KILLX2 := 16
-MODE_INVISIBLE := 17
-MODE_HARDDROP := 18
-MODE_SPEED_TEST := 19
-MODE_SCORE_DISPLAY := 20
-MODE_HZ_DISPLAY := 21
-MODE_INPUT_DISPLAY := 22
-MODE_DISABLE_FLASH := 23
-MODE_DISABLE_PAUSE := 24
-MODE_GOOFY := 25
-MODE_DEBUG := 26
-MODE_LINECAP := 27
-MODE_DASONLY := 28
-MODE_QUAL := 29
-MODE_PAL := 30
-
-MODE_QUANTITY := 31
-MODE_GAME_QUANTITY := 19
+.enum
+MODE_TETRIS
+MODE_TSPINS
+MODE_SEED
+MODE_PARITY
+MODE_PACE
+MODE_PRESETS
+MODE_TYPEB
+MODE_FLOOR
+MODE_CRUNCH
+MODE_TAP
+MODE_TRANSITION
+MODE_TAPQTY
+MODE_CHECKERBOARD
+MODE_GARBAGE
+MODE_DROUGHT
+MODE_DAS
+MODE_KILLX2
+MODE_INVISIBLE
+MODE_HARDDROP
+MODE_SPEED_TEST
+MODE_SCORE_DISPLAY
+MODE_HZ_DISPLAY
+MODE_INPUT_DISPLAY
+MODE_DISABLE_FLASH
+MODE_DISABLE_PAUSE
+MODE_GOOFY
+MODE_DEBUG
+MODE_LINECAP
+MODE_DASONLY
+MODE_QUAL
+MODE_PAL
+.endenum
+
+MODE_QUANTITY = MODE_PAL + 1
+MODE_GAME_QUANTITY = MODE_SPEED_TEST
```

Comment:
I think this is better as `MODE_HARDDROP + 1` perhaps

Reply:
That does make it more clear!  Made the adjustment and validated sha1sum is the same.